analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -190,15 +192,38 @@ class SpdxDocumentFile(
             Package(
                 id = pkg.toIdentifier(),
                 declaredLicenses = sortedSetOf(pkg.licenseDeclared),
-                concludedLicense = pkg.licenseConcluded.toSpdx(),
+                concludedLicense = buildConcludedLicense(pkg),
                 description = packageDescription,
                 homepageUrl = pkg.homepage.mapNotPresentToEmpty(),
                 binaryArtifact = RemoteArtifact.EMPTY, // TODO: Use "downloadLocation" or "externalRefs"?
-                sourceArtifact = RemoteArtifact.EMPTY, // TODO: Use "sourceInfo" or "externalRefs"?
+                sourceArtifact = buildSourceArtifact(pkg), // TODO: Use "sourceInfo" or "externalRefs"?
                 vcs = VersionControlSystem.forDirectory(packageDir)?.getInfo() ?: VcsInfo.EMPTY
             )
         }
 
         return listOf(ProjectAnalyzerResult(project, packages))
     }
+
+    /**
+     * Return null in case of "NONE" or "NOASSERTION" to fit with the standard process ( concluded manage by curations )
+     */
+    private fun buildConcludedLicense(pkg: SpdxPackage): SpdxExpression? {
+        return when {
+            pkg.licenseConcluded == SpdxConstants.NONE -> null
+            pkg.licenseConcluded == SpdxConstants.NOASSERTION -> null
+            else -> pkg.licenseConcluded.toSpdx()
+        }
+    }
+
+    /**
+     * Return a [RemoteArtifact] created with downloadLocation of the given package [SpdxPackage]
+     * if it's a file location or return en Empty RemoteArtifact
+     */
+    private fun buildSourceArtifact(pkg: SpdxPackage): RemoteArtifact {
```

Comment:
The addition of this is a different topic and should be done in a separate commit with a separate commit message explaining the details.

Reply:
Ok, I'll be careful next time

Reply:
No worries, this can happen, and we're one of those project which are really strict about this, also see https://github.com/oss-review-toolkit/ort/blob/master/CONTRIBUTING.md#git-history.